certified-connectors/intelliHR/apiDefinition.swagger.json

```diff
@@ -0,0 +1,7967 @@
+            "in": "query",
+            "required": false,
+            "type": "integer",
+            "format": "int32",
+            "minimum": 1,
+            "exclusiveMinimum": false,
+            "description": "The page number",
+            "x-ms-summary": "The page number"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/ListofEmploymentConditions"
+            },
+            "headers": {}
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {
+              "$ref": "#/definitions/InternalServerError"
+            },
+            "examples": {
+              "application/json": {
+                "value": {
+                  "name": "Internal Server Error",
+                  "message": "An internal server error occurred and has been recorded. Please try again later. If this does not resolve your issue, please contact support@intellihr.co for support and guidance.",
+                  "code": 500,
+                  "errors": {}
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/employment-conditions/{id}": {
+      "get": {
+        "description": "Returns a single employment condition.",
+        "summary": "Find an Employment Condition by ID",
+        "tags": [
+          "Employment Conditions"
+        ],
+        "operationId": "FindanEmploymentConditionbyID",
+        "deprecated": false,
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource identifier string",
+            "x-ms-summary": "The employment condition ID",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SingleEmploymentCondition"
+            },
+            "headers": {}
+          },
+          "500": {
+            "description": "Internal Server Error",
+            "schema": {
+              "$ref": "#/definitions/InternalServerError"
+            },
+            "examples": {
+              "application/json": {
+                "value": {
+                  "name": "Internal Server Error",
+                  "message": "An internal server error occurred and has been recorded. Please try again later. If this does not resolve your issue, please contact support@intellihr.co for support and guidance.",
+                  "code": 500,
+                  "errors": {}
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/job-end/{id}": {
+      "patch": {
+        "responses": {
+          "default": {
```

Comment:
@wolskis We recommend you to replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Reply:
Amended